docs/src/main/sphinx/installation/deployment.md

```diff
@@ -39,10 +39,10 @@ Trino requires a 64-bit version of Java 17, with a minimum required version of 1
 Earlier major versions such as Java 8 or Java 11 do not work.
 Newer major versions such as Java 18 or 19, are not supported -- they may work, but are not tested.
 
-We recommend using [Azul Zulu](https://www.azul.com/downloads/zulu-community/)
-as the JDK for Trino, as Trino is tested against that distribution.
-Zulu is also the JDK used by the
-[Trino Docker image](https://hub.docker.com/r/trinodb/trino).
+We recommend using the Eclipse Temurin OpenJDK distribution from
+[Adoptium](https://adoptium.net/) as the JDK for Trino, as Trino is tested
```

Comment:
This is partially true. Product tests environments are tested with Zulu distribution still. We cannot yet move to Temurin there as it's now linked against glibc 2.14 which is newer than glibc 2.12 used in Centos6-based images (HDP2.6, CDH5). We are using Temurin for both maven executed checks and Docker images that we publish.

We should update the recommended JDK in the RPM preinstall script as well.

Reply:
So should we wait with this doc update @wendigo and @electrum ? Also .. are you updating the RPM script and the product tests @wendigo?

Reply:
@mosabua We are in the process of updating product tests environments but it will take some time. i'll defer the decision to merge this change to @electrum